editions/tw5.com/tiddlywiki.info

```diff
@@ -5,6 +5,7 @@
 		"tiddlywiki/confetti",
 		"tiddlywiki/dynannotate",
 		"tiddlywiki/internals",
+		"tiddlywiki/freelinks",
```

Comment:
This plugin will need to be removed once you are sure, that everything works as expected.